client/src/components/Background.tsx

```diff
@@ -1,24 +1,4 @@
 const Background = () => {
-  const splashStyles = {
```

Comment:
Can we keep this around actually, maybe put the trump thing in a new component and rename this one GradientBackground instead